docs/using_other_models.rst

```diff
@@ -54,12 +54,12 @@ usage with
 .. code-block:: python
 
     panels = [
-        AutocompletePanel('external_link', page_type='app_label.Link'),
+        AutocompletePanel('external_link', target_model='app_label.Link'),
     ]
 
 .. note::
     :doc:`wagtail-autocomplete <index>` does not offer two separate edit
     handlers like Wagtail does for Page and Snippet. As such,
     ``AutocompletePanel`` does not support the ``snippet_type`` kwarg that
     :class:`~wagtail:wagtail.wagtailsnippets.edit_handlers.SnippetChooserPanel`
-    does. Instead, ``page_type`` should be used.
```

Comment:
I think this note is actually unnecessary with the keyword change. Let's take it out entirely.